src/core/unicode.js

```diff
@@ -1630,8 +1630,29 @@ function reverseIfRtl(chars) {
   return s;
 }
 
+function keepUnifiedIdeograph(str, normalizedUnicodesMap) {
+  // a character id map to a unicode. replace it if it is not unified
+  if (str.length === 1) {
+    if (normalizedUnicodesMap[str]) {
+      return normalizedUnicodesMap[str];
+    }
+    return str;
+  }
+  // a character id map to multiple unicode. remove not unified unicodes
+  var outStr = '';
+  for (var i = 0; i < str.length; i++) {
```

Comment:
`var` should be `let` and the string length can be cached, so this can become `for (let i = 0, ii = str.length; i < ii; i++) {`